exporter/otlpexporter/config.go

```diff
@@ -22,10 +22,31 @@ type Config struct {
 }
 
 func (c *Config) Validate() error {
+	if err := c.validateEndpointNotEmpty(); err != nil {
+		return err
+	}
+	if err := c.validateEndpointHasPort(); err != nil {
+		return err
+	}
```

Comment:
The logic inside these functions is simple enough that we can keep everything inside `Validate`.

Reply:
Addressed in ec2b9d32